healthcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+tips=$(dig +short tasks.$SERVICE_NAME)
+nbt=$(echo $tips | wc -w)
+
+# we do not have min seeds number yet
+# peacefully exit to allow swarm to add new node to DNS list (service.$SERVICE_NAME) 
+# see #1462
+[[ $nbt -lt ${MIN_SEEDS_COUNT} ]] && exit 0
```

Comment:
in case we have a single replica and the SERVICE_NAME and MIN_SEEDS_COUNT are not set, the script will always exit 0, even if etcd is failing.